.github/workflows/docker-build.yml

```diff
@@ -208,7 +215,8 @@ jobs:
       - name: Check out the code at a specific ref
         uses: actions/checkout@v4
         with:
-          ref: ${{ needs.get-version.outputs.version || github.ref }}
+          ref: ${{ inputs.nightly_tag_main || github.ref }}
+          #ref: ${{ needs.get-version.outputs.version && format('v{0}', needs.get-version.outputs.version) || github.ref }}
```

Comment:
There's been significant changes (simplifications, hopefully) to docker-build in terms of these inputs

Reply:
in an unmerged pr